_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,121 @@
+
+Let's look at an example of an alternate release process that incorporates these three pillars.
+
+<div style = "display: block; width: inherit; height: 400px; text-align: center;">
+    <a href="/ITStrategy/assets/images/Feature-Intake-to-Release-BPM-Alternate.png" style="display: block;">
+        <img src = "/ITStrategy/assets/images/Feature-Intake-to-Release-BPM-Alternate.png"
+        alt = "Map of alternate release process including which incorporates DevOps methodologies to enable frequent releases. Functions are encapsulated within a single product teams and interactions with central operational teams are through automated checks and tests in incorporated in the build pipeline"
+        style="width: 550px; height: 399px;"
+        />
+    </a>
+    <small> A possible alternate release process ( You can click to expand this image )</small>
+</div>
+<br/>
+
+Already we can see this is subjectively a simpler process to digest and understand. In this process, two weeks of work take a day or less to release. All the functions are encapsulated within one product team. Interaction with functions such as IT Security, Accessibility and Testing Services is through automated checks and processes incorporated into the build. These central operational teams are instead tasked with continually addressing bottlenecks and championing improvements rather then solely a fire fighting role. An incident response role still exists, things will go wrong sometimes regardless of any process or methodology taken. The outcome of these incidents however is not to add an unrealistic burden to operational staff to manually ensure something doesn't happen again, rather fixing the root of the issue by improving the process and incorporating automated checks to ensure that this incident never happens again.
+
+This drastically reduces lead time, we are no longer playing a game of catch up. We are able to put our focus on delivering the best possible services to our citizens rather than navigating a process. Developers are able to focus on their work of adding features and addressing technical debt rather than figuring out a complex process that takes longer than the actual work itself. Operational staff are not over burdened by being constantly in fire fighting mode and can instead focus on innovating and improving the process to make it even more efficient and secure. Management is able to quickly deliver and exceed their commitments. Risk is reduced by ensuring that every build of an application or service works as expected and is automatically compliant to our security and accessibility commitments that we are bound to by law.
+
+A better release process is possible in our organization and necessary to be able to respond to increasingly changing environment. Continuing with our current process is a significant risk and we must work together as the technology organization to address this large organizational debt.
```

Comment:
This you may want to address in the Napoleon story. I.e. with DevOps, we can make rapid and frequent small changes, reducing the need to build large upfront analysis and advance planning  (that PMLC gating process). Such large upfront analysis and advance planning is a natural immune system reaction from the organization that seeks control and predictability (probably due to, in the pre-DevOps days, it was very timely and difficult to make an IT change). 

This to drive the point that DevOps is not just an execution phase of a project that developers use. It should be business as well.

Reply:
I guess but it's a chicken or an egg scenario. Do we have a large release process because we have large upfront analysis or vice versa. I suspect the answer is somewhere in between but the point of this blog was to talk about the release process